nxcomp/src/Loop.cpp

```diff
@@ -11733,14 +11734,11 @@ int SetPorts()
 	  << logofs_flush;
 #endif
 
-  useSmbSocket = 0;
-  if (smbPort.enabled()) {
-    if (control -> ProxyMode == proxy_client) {
-      auxPort.setDefaultTCPPort(DEFAULT_NX_SMB_PORT_OFFSET + proxyPort);
-      useAuxSocket = 1;
-    }
-    else
-      auxPort.setDefaultTCPPort(139);
+  if (control -> ProxyMode == proxy_client) {
+    smbPort.setDefaultTCPPort(DEFAULT_NX_SMB_PORT_OFFSET + proxyPort);
+    useSmbSocket = smbPort.enabled();
+  } else {
+    smbPort.setDefaultTCPPort(139);
```

Comment:
Looking at this... Shouldn't we move to port 445 for this? Rather do it now while refactoring everything?

Reply:
Hmm... can't that cause issues with backwards compatibility?

Reply:
Commit message fixed